src/applications/_mock-form-ae-design-patterns/patterns/pattern2/post-study/UneditableData.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { InfoSection } from '../../../shared/components/InfoSection';
+import { maskSSN } from '../../../utils/helpers/general';
+
+const AdditionalInfoContent = () => {
+  return (
+    <div>
+      <p className="vads-u-margin-top--0">
+        To protect your personal information, we don’t allow online changes to
+        your name, date of birth, or Social Security number. If you need to
+        change this information, call Veterans Benefits Assistance at{' '}
+        <va-telephone contact="8008271000" />. We’re here Monday through Friday,
+        between 8:00 a.m. and 9:00 p.m. ET.
+      </p>
+
+      <va-link
+        text="Find instructions for how to change your legal name"
+        href="/resources/how-to-change-your-legal-name-on-file-with-va/"
+      />
+    </div>
+  );
+};
+
+const UneditableData = () => {
```

Comment:
I'm still thinking about if it would be possible to show this or any of the alerts side by side to get peoples read of them while being able to visually compare them quickly. Maybe we could make the layout full width for this page only to use flexbox and see how they work as columns?

Just a thought for further experimentation.